Makefile

```diff
@@ -89,36 +89,48 @@ ifeq ($(TARGET_ARCH),x64)
 	TARGET_ARCH_DEBIAN = amd64
 endif
 
-TEMPORARY_DIRECTORY = release/.tmp
```

Comment:
How do you feel about making the whole `release` prefixes configurable by variables? I imagine we could ship this Makefile for other users to `include`, in which case we could make the "build" directory configurable.